cache/blobs.go

```diff
@@ -422,7 +423,7 @@ func ensureCompression(ctx context.Context, ref *immutableRef, comp compression.
 		}
 
 		// Resolve converters
-		layerConvertFunc, err := getConverter(ctx, ref.cm.ContentStore, desc, comp)
+		layerConvertFunc, err := converter.New(ctx, ref.cm.ContentStore, desc, comp)
```

Comment:
can be follow-up: This `New` pattern doesn't work with function anymore and should probably return a struct/interface.

Reply:
Why not work?

Reply:
> `New` implies a new instance of a struct.

I'm not sure.
I thought it can return any.